joern-cli/frontends/php2cpg/src/test/scala/io/joern/php2cpg/querying/CallTests.scala

```diff
@@ -53,12 +72,19 @@ class CallTests extends PhpCode2CpgFixture {
       fooCall.name shouldBe "foo"
       fooCall.methodFullName shouldBe s"foo"
       fooCall.signature shouldBe s"${Defines.UnresolvedSignature}(1)"
-      fooCall.receiver.isEmpty shouldBe true
-      fooCall.dispatchType shouldBe DispatchTypes.STATIC_DISPATCH
+      fooCall.receiver.isEmpty shouldBe false
+      fooCall.receiver.foreach { recvNode =>
+        val recv = recvNode.asInstanceOf[Identifier]
+        recv.code shouldBe "foo"
+        recv.name shouldBe "foo"
+      }
+
+      fooCall.dispatchType shouldBe DispatchTypes.DYNAMIC_DISPATCH
```

Comment:
The call to `foo` should be static. The only thing to do is check the summary in order to find out whether this references a method from the global or local namespace.